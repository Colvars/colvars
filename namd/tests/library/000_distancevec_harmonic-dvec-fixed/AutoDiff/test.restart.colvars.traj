# step         one                                                                       fa_one                                                                    
          20   ( -1.00012595362068e+00 ,  6.81665091863780e-01 ,  2.98047387611607e+00 ) (  4.00050381448271e-03 , -2.72666036745512e-03 , -1.19218955044643e-02 ) 
          21   ( -9.99138074893485e-01 ,  6.78576393089195e-01 ,  2.97902552991010e+00 ) (  3.99655229957394e-03 , -2.71430557235678e-03 , -1.19161021196404e-02 ) 
          22   ( -9.98651128006031e-01 ,  6.75203199276404e-01 ,  2.97741796069013e+00 ) (  3.99460451202413e-03 , -2.70081279710561e-03 , -1.19096718427605e-02 ) 
          23   ( -9.98610825708839e-01 ,  6.71594275118549e-01 ,  2.97578645094022e+00 ) (  3.99444330283536e-03 , -2.68637710047419e-03 , -1.19031458037609e-02 ) 
          24   ( -9.98946204647680e-01 ,  6.67806005306121e-01 ,  2.97425809124110e+00 ) (  3.99578481859072e-03 , -2.67122402122448e-03 , -1.18970323649644e-02 ) 
          25   ( -9.99574199242226e-01 ,  6.63901368166392e-01 ,  2.97294412310483e+00 ) (  3.99829679696891e-03 , -2.65560547266557e-03 , -1.18917764924193e-02 ) 
          26   ( -1.00040433184298e+00 ,  6.59947725601624e-01 ,  2.97193439791086e+00 ) (  4.00161732737193e-03 , -2.63979090240649e-03 , -1.18877375916435e-02 ) 
          27   ( -1.00134314986632e+00 ,  6.56013782607379e-01 ,  2.97129431690302e+00 ) (  4.00537259946528e-03 , -2.62405513042951e-03 , -1.18851772676121e-02 ) 
          28   ( -1.00229812026062e+00 ,  6.52166332643251e-01 ,  2.97106411683570e+00 ) (  4.00919248104249e-03 , -2.60866533057301e-03 , -1.18842564673428e-02 ) 
          29   ( -1.00318079820382e+00 ,  6.48467421471618e-01 ,  2.97125993940365e+00 ) (  4.01272319281527e-03 , -2.59386968588647e-03 , -1.18850397576146e-02 ) 
          30   ( -1.00390922101371e+00 ,  6.44972467521952e-01 ,  2.97187614715496e+00 ) (  4.01563688405484e-03 , -2.57988987008781e-03 , -1.18875045886199e-02 ) 
          31   ( -1.00440959005691e+00 ,  6.41729568368808e-01 ,  2.97288843458314e+00 ) (  4.01763836022764e-03 , -2.56691827347523e-03 , -1.18915537383326e-02 ) 
          32   ( -1.00461738044422e+00 ,  6.38779886606265e-01 ,  2.97425757145878e+00 ) (  4.01846952177690e-03 , -2.55511954642506e-03 , -1.18970302858351e-02 ) 
          33   ( -1.00447805084808e+00 ,  6.36158624654867e-01 ,  2.97593366048238e+00 ) (  4.01791220339232e-03 , -2.54463449861947e-03 , -1.19037346419295e-02 ) 
          34   ( -1.00394748630005e+00 ,  6.33895972963095e-01 ,  2.97786077691918e+00 ) (  4.01578994520019e-03 , -2.53558389185238e-03 , -1.19114431076767e-02 ) 
          35   ( -1.00299228530624e+00 ,  6.32017486547197e-01 ,  2.97998162119086e+00 ) (  4.01196914122495e-03 , -2.52806994618879e-03 , -1.19199264847635e-02 ) 
          36   ( -1.00158989063031e+00 ,  6.30543620217316e-01 ,  2.98224176226361e+00 ) (  4.00635956252122e-03 , -2.52217448086926e-03 , -1.19289670490544e-02 ) 
          37   ( -9.99728647879413e-01 ,  6.29488492575348e-01 ,  2.98459308226489e+00 ) (  3.99891459151765e-03 , -2.51795397030139e-03 , -1.19383723290595e-02 ) 
          38   ( -9.97407895568871e-01 ,  6.28858255745633e-01 ,  2.98699626416867e+00 ) (  3.98963158227548e-03 , -2.51543302298253e-03 , -1.19479850566747e-02 ) 
          39   ( -9.94638189030939e-01 ,  6.28649565358814e-01 ,  2.98942235138835e+00 ) (  3.97855275612376e-03 , -2.51459826143525e-03 , -1.19576894055534e-02 ) 
          40   ( -9.91441715714853e-01 ,  6.28848618963486e-01 ,  2.99185347903944e+00 ) (  3.96576686285941e-03 , -2.51539447585394e-03 , -1.19674139161578e-02 ) 
