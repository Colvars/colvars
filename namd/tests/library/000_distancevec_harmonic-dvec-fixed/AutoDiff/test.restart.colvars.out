colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # indexFile = "index.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # sourceTclFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "one"
colvars:   Initializing a new "distanceVec" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group1"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group1".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 4 atoms requested: total mass = 54.028, total charge = -0.72.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group2"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group2".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 4 atoms requested: total mass = 54.028, total charge = -0.4.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.5
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { one }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { ( 0 , 0 , 0 ) }
colvars:   # targetCenters = { ( 0 , 0 , 0 ) } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 0.001
colvars:   # decoupling = off [default]
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "one" will be rescaled to 0.004 according to the specified width (0.5).
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 8 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - Colvars module:
colvars: - Colvars-NAMD interface:
colvars: - Harmonic colvar bias implementation:
colvars: - Optimal rotation via flexible fitting:
colvars: - distance colvar component:
colvars: - distanceVec colvar component (derived from distance):
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - NAMD engine:
colvars: - Scalable center-of-mass computation (NAMD):
colvars:   Phillips2020 https://doi.org/10.1063/5.0014475
colvars: 
colvars: updating target temperature (T = 0 K).
colvars: ----------------------------------------------------------------------
colvars: Loading state from file "test.colvars.state".
colvars:   Restarting collective variable "one" from value: ( -1.00013 , 0.681665 , 2.98047 )
colvars:   Restarting harmonic bias "harmonic1" from step number 20.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 8 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: updating target temperature (T = 0 K).
colvars: Re-initialized atom group for variable "one":0/0. 4 atoms: total mass = 54.028, total charge = -0.72.
colvars: Re-initialized atom group for variable "one":0/1. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Saving collective variables state to "test.restart.colvars.state".
